evaluators/rank/FScoreEvaluator/__init__.py

```diff
@@ -0,0 +1,44 @@
+from typing import Sequence, Any, Optional
+
+from jina.executors.evaluators.rank import BaseRankingEvaluator
+
+
+class FScoreEvaluator(BaseRankingEvaluator):
+    """
+    :class:`FScoreEvaluator` Gives the f score of a search system result. (https://en.wikipedia.org/wiki/F-score)
+    :param eval_at: the point at which precision and recall are computed, if None give, will consider all the input to evaluate
```

Comment:
can we add a blank line between class description and `param` section?

Reply:
Can u recheck?